### ISSUE TITLE
Confused about evaluation results for net_rain100l on RealRain-1K

### ISSUE THREAD

**Issue by FeiSong123** [GuangluDong0728/CSUD-Unsupervised-Deraining-CVPR2025#3]
Hi authors,

I encountered inconsistent results when evaluating the net_rain100l model on the RealRain-1K dataset (tested on the validation subset with images cropped to 512x512 input size). Could you clarify if my evaluation settings might be incorrect?

For reference, could you share the official performance metrics (e.g., PSNR/SSIM) obtained under your testing pipeline?

**Comment by GuangluDong0728** [comment]
> RealRain-1K dataset

**Comment by GuangluDong0728** [comment]
> Hi authors,
> 
> I encountered inconsistent results when evaluating the net_rain100l model on the RealRain-1K dataset (tested on the validation subset with images cropped to 512x512 input size). Could you clarify if my evaluation settings might be incorrect?
> 
> For reference, could you share the official performance metrics (e.g., PSNR/SSIM) obtained under your testing pipeline?

Hello! Thank you for your message. Regarding your question, I think there are two reasons as follows:
First of all, you shouldn't crop or resize the images to 512×512, as this will significantly affect the evaluation of the metrics. Did you do this because of insufficient GPU memory? Secondly, the RealRain - 1K dataset is divided into two sub - datasets, RealRain - 1K - L and RealRain - 1K - H. Each of these sub - datasets is further split into three subsets: Train, Test, and Validation. We use the train set for training and the test set for testing. All the results reported in the paper are based on the test set. Could it be that you've conducted the tests on the Validation set?  If you have any other questions, you're welcome to ask. I'm more than happy to answer them for you. For the convenience of communication, you can also add my WeChat ID: A2510898651.
